libraries/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ set( LIBRARIES
    lib-menus
    lib-note-track
    lib-viewport
+   lib-music-information-retrieval
```

Comment:
I'm doing some after-the-fact review of this because I need the background to understand #5791.

I note the first commit in the sequence didn't build (test compilation failed)